tools/ros2_dependencies.repos

```diff
@@ -2,7 +2,7 @@ repositories:
   ros-planning/navigation2:
     type: git
     url: https://github.com/ros-planning/navigation2.git
-    version: master
+    version: main
```

Comment:
Interesting, do you know if this is this temporary or not?

Reply:
This should be permanent, I think other packages will be making this change as well AFAIK.